src/main/java/org/broadinstitute/hellbender/tools/funcotator/dataSources/DataSourceUtils.java

```diff
@@ -283,16 +360,52 @@ public static boolean isValidDirectory(final Path p) {
         return dataSourceFactories;
     }
 
+    private static FeatureInput<? extends Feature> createAndRegisterFeatureInputs(final Path dataSourceFile,
+                                                                                  final Properties dataSourceProperties,
+                                                                                  final GATKTool funcotatorToolInstance,
+                                                                                  final int lookaheadFeatureCachingInBp,
+                                                                                  final Class<? extends Feature> featureType) {
+        Utils.nonNull(dataSourceFile);
+        Utils.nonNull(dataSourceProperties);
+
+        final String name      = dataSourceProperties.getProperty(CONFIG_FILE_FIELD_NAME_NAME);
+        final String sourceFile = dataSourceFile.resolveSibling(dataSourceProperties.getProperty(CONFIG_FILE_FIELD_NAME_SRC_FILE)).toString();
+
+        // Get feature inputs by creating them with the funcotator tool instance itself:
```

Comment:
Add note that this has the side effect of registering the `FeatureInputs` with the engine, so that they can be later queried.

Reply:
Fixed.